ibis/backends/dask/tests/execution/test_arrays.py

```diff
@@ -34,22 +44,32 @@ def test_array_length(t, df):
 
 
 def test_array_length_scalar(client):
-    raw_value = [1, 2, 4]
-    value = ibis.literal(raw_value)
+    raw_value = np.array([1, 2, 4])
+    value = ibis.array(raw_value)
     expr = value.length()
     result = client.execute(expr)
     expected = len(raw_value)
     assert result == expected
 
 
+@pytest.mark.xfail(
```

Comment:
I'm surprised that this is now failing and don't see anything in the tracker for this. Can you give more detail?

Reply:
I don't think this was being tested anywhere actually (here I renamed the original `test_array_collect` to `test_array_collect_grouped` and created a new `test_array_collect` that tests `.collect()` without a groupby — this is the functionality that is failing)

I think it fails due to the `Summarize` aggcontext's implementation not being fully compatible with the Dask backend. It works if the `function` passed to `Summarize.agg` is a string, which hits this code path:
https://github.com/ibis-project/ibis/blob/6f6f0ff77899afbce694a0d25cf3edd0d297f33c/ibis/backends/pandas/aggcontext.py#L358-L359

However, if `function` is a function, it hits this code path, which fails because `dd.Series` seems to not have `.agg` implemented:
https://github.com/ibis-project/ibis/blob/6f6f0ff77899afbce694a0d25cf3edd0d297f33c/ibis/backends/pandas/aggcontext.py#L366